hrjobcontract/js/src/job-contract/controllers/contract-list.html

```diff
@@ -51,7 +51,7 @@ <h3 class="header-bar header-bar-danger" ng-show="contractPast.length">Past Cont
         </div>
       </div>
       <div class="row">
-        <div class="col-xs-12"><hr></div>
+        <div class="col-xs-12"><hr /></div>
```

Comment:
The closing slash is not needed:
https://www.w3.org/TR/html/syntax.html#void-elements